app/poros/food.rb

```diff
@@ -0,0 +1,9 @@
+class Food
+  attr_reader :code, :description, :brand_owner, :ingredients
+  def initialize(data)
+    @code = data[:gtinUpc]
+    @description = data[:description]
+    @code = data[:brandOwner]
+    @code = data[:ingredients]
```

Comment:
Watch your instance variables. This was probably just a copy/paste oversight but these will compete with each other as the same ivar can't have multiple values.